From 869de265e2167891be72c99246505a5c001edc52 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 4 Dec 2014 10:06:52 +0800
Subject: [PATCH 1051/1543] MLK-9955-7 arm: dts: imx6sx: add mu support

commit d8d354f777c1f9921c8f5b5acafb065e6e739cd1 from
git://git.freescale.com/imx/linux-2.6-imx.git

add MU support for i.MX6SX.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 8f696f9..7234248 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1320,6 +1320,13 @@
 				status = "okay";
 			};
 
+			mu: mu@02294000 { /* mu */
+				compatible = "fsl,imx6sx-mu";
+				reg = <0x02294000 0x4000>;
+				interrupts = <0 90 0x04>;
+				status = "okay";
+			};
+
 			uart6: serial@022a0000 {
 				compatible = "fsl,imx6sx-uart", "fsl,imx21-uart";
 				reg = <0x022a0000 0x4000>;
-- 
1.7.5.4

